tests/pykafka/test_producer.py

```diff
@@ -36,7 +35,7 @@ def test_produce(self):
 
         # set a timeout so we don't wait forever if we break producer code
         message = self.consumer.consume()
-        self.assertTrue(message.value == payload)
+        assert message.value == payload
```

Comment:
This was me cleaning up the tests so that they make sense in failures.  The error message looked like this before:

```
False is not true
```

Instead of:

```
b'test-data' != 'test-data'
```